src/index.js

```diff
@@ -18,14 +18,14 @@ import { verifyAuthority } from './middleware/verifyAuth.js';
 import { validateParams } from './middleware/validation.js';
 import { endpointAvailable } from './middleware/endpoint.js';
 import cookieParser from 'cookie-parser';
-import Configuration from './models/configuration.model.js';
-import db from './db/database.js';
+import { models } from './models/models.js';
+import { sequelize } from './db/database.js';
 import { verifyAdmin } from './middleware/verifyAdmin.js';
 import swaggerJSDoc from 'swagger-jsdoc';
 import swaggerUi from 'swagger-ui-express';
 
 // Check if we are in a test environment
-const isTestEnvironment = process.env.NODE_ENV === 'test';
+const isTestEnvironment = import.meta.env?.VITEST;
```

Comment:
```suggestion
const isTestEnvironment = !!import.meta.env?.VITEST;
```
Given the name we used for this variable, it makes sense to forcefully cast it into a boolean in my opinion.